src/main/java/com/easypost/model/EndShipper.java

```diff
@@ -0,0 +1,117 @@
+package com.easypost.model;
+
+import com.easypost.EasyPost;
+import com.easypost.exception.EasyPostException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class EndShipper extends BaseAddress {
+    /**
+     * Create EndShipper object from parameter map.
+     *
+     * @param params Map of end shipper parameters.
```

Comment:
```suggestion
     * @param params Map of EndShipper parameters.
```
This also occurs in other docstrings, but won't block it

Reply:
Fixed, thanks for the callout